docs/modules/ROOT/pages/python-client-getting-started.adoc

```diff
@@ -0,0 +1,195 @@
+(sync) method calls, which is better for getting started. For async calls, please check our https://hazelcast.readthedocs.io/en/stable/#usage[API documentation].
+
+With these lines, client adds data to `cities` map. The first parameter is the key of the entry, the second one is the value as mentioned above.
+
+[source, python]
+----
+cities_map.set(1, "London")
+cities_map.set(2, "New York")
+cities_map.set(3, "Tokyo")
+----
+
+Then, we get the values with the `entry_set()` method and iterate over the set.
+
+[source, python]
+----
+entries = cities_map.entry_set()
+
+for key, value in entries:
+    print(f"{key} -> {value}")
+----
+
+Finally, `client.shutdown()` terminates our client and release its resources.
+
+== Add a Listener to the Map
+
+You can add an entry listener using the `add_entry_listener()` method available on the map proxy object.
+This will allow you to listen to certain events that happen in the map across the cluster.
+
+The first argument to the `add_entry_listener()` method is `includeValue`.
+This boolean parameter, if set to true, ensures the entry event contains the entry value.
+
+The second argument to the `add_entry_listener()` method is an object that is used to define listeners.
+In this example, we register listeners for the `added`, `removed` and `updated` events.
+
+This enables your code to listen to map events of that particular map.
+
+[source, python]
+----
+import hazelcast
+
+def entry_added(event):
+    print(f"Entry added with key: {event.key}, value: {event.value}")
+
+
+def entry_removed(event):
+    print(f"Entry removed with key: {event.key}")
+
+
+def entry_updated(event):
+    print(f"Entry updated with key: {event.key}, old value: {event.old_value}, new value: {event.value}")
+
+client = hazelcast.HazelcastClient()
+
+cities_map = client.get_map('cities').blocking()
+cities_map.add_entry_listener(
+    include_value=True, added_func=entry_added, removed_func=entry_removed, updated_func=entry_updated
+)
+
+cities_map.clear()
+cities_map.set(1, "London")
+cities_map.set(2, "New York")
+cities_map.set(3, "Tokyo")
+
+cities_map.remove(1)
+cities_map.replace(2, "Paris")
+
+cities_map.(1)
+
+entries = cities_map.entry_set()
+
+for key, value in entries:
+    print(f"{key} -> {value}")
+
+client.shutdown()
+----
+
+First, the map is cleared to fire events even if there are some entries in the map. Then, entries are added, and they are logged. After that, we remove one of the entries and update the other one. Then, we log the entries again.
+
+The output is as follows.
+
+[source, bash]
+----
+Entry added with key: 1, value: London
+Entry added with key: 2, value: New York
+Entry added with key: 3, value: Tokyo
+Entry removed with key: 1
+Entry updated with key: 2, old value: New York, new value: Paris
+2 -> Paris
+3 -> Tokyo
+----
+
+The value of the first entry becomes `None` since it is removed.
```

Comment:
Actually there is no such entry any more, hence I am not sure if we should mention about a value of a non-existent entry.